frontend/src/actions/transactions.js

```diff
@@ -0,0 +1,81 @@
+import myApi from './api'
+import uuid from 'uuid';
+
+export const addTransaction = (
+    {
+        id = uuid(),
+        description = '',
+        note = '',
+        amount = 0,
+        createdAt = 0
+    } = {}
+) => ({
+    type: 'ADD_TRANSACTION',
+    transaction: {
+        id,
+        description,
+        note,
+        amount,
+        createdAt
+    }
+})
+
+export const startAddTransaction = (description, note, amount, createdAt) => {
+    const id = uuid()
+    const action = addTransaction({
+        id,
```

Comment:
why id is created there? I think backend should return id from db (missing query to the backend after creating or id should be attached to response in post query (?))